### ISSUE TITLE
Fatal error in get_fund_historical_data() for certain combinations of start_date and end_date

### ISSUE THREAD

**Issue by BoxTrot** [alvarobartt/investpy#262]
When the start date and the end_date are more than 19 years apart, the function attempts to construct a new end_date that is less than 19 years apart using the start_date. However, if the start_date is on the last day of the month, the function constructs an invalid date at 465 in funds.py by adding 1 to the day number (excerpt below; problem line highlighted).

This is easily reproducible by calling the function as follows:

```python
import investpy
investpy.get_fund_historical_data("Amundi Funds Ii - U.s. Pioneer Fund C Usd Nd", "luxembourg", from_date="30/11/2001", to_date="06/01/2021")
```

And can be fixed either by removing the "+ 1" in line 465 or, using timedelta from datetime, replacing line 465 with:

```python
start_date = start_date.replace(year=start_date.year + 19) + timedelta(days=1)
```

Excerpt of problematic code from funds.py:

```python
flag = True

while flag is True:
	diff = end_date.year - start_date.year

	if diff > 19:
		obj = {
			'start': start_date.strftime('%m/%d/%Y'),
			'end': start_date.replace(year=start_date.year + 19).strftime('%m/%d/%Y'),
		}

		date_interval['intervals'].append(obj)

		start_date = start_date.replace(year=start_date.year + 19, day=start_date.day + 1) # Leads to a ValueError: day is out of range for month
	else:
		obj = {
			'start': start_date.strftime('%m/%d/%Y'),
			'end': end_date.strftime('%m/%d/%Y'),
		}

		date_interval['intervals'].append(obj)

		flag = False
```

**Comment by alvarobartt** [comment]
Thanks for reporting this issue @BoxTrot! I'll be fixing it this week so that best-case scenario this weekend I'll be releasing the 1.0.2 `investpy` version including this bug-fix and all the ones already solved!

I'll let you know whenever this is ready! 🔥

**Comment by alvarobartt** [comment]
Thanks a lot @BoxTrot! 🔥 This has already been solved and will be available in the `v1.0.3` release in a couple of days.
